server/app/app.test.js

```diff
@@ -1,8 +1,15 @@
 const request = require('supertest');
 const app = require('./app');
 const db = require('../data/db');
+const { server } = require('../../src/mocks/server');
 
 describe('Endpoints', () => {
+  beforeAll(() => {
+    server.close();
+  });
+  afterAll(() => {
+    server.listen();
+  });
```

Comment:
This is required to prevent mock service worker from intercepting the traffic between the tests and the endpoints that are being tested